spec/features/item_view_metadata_spec.rb

```diff
@@ -1,17 +1,20 @@
 require 'spec_helper'
 
 feature 'Item view metadata' do
-  before do
-    @current_user = double(
+  let(:current_user) do
```

Comment:
I'm concerned that the instance variable approach is also part of the problem here. Setting an instance variable is not an rspec convention (that I can find) and hope to move away from this practice:

http://betterspecs.org/#let
http://stackoverflow.com/questions/5359558/when-to-use-rspec-let/5359979#5359979